impl/Cargo.toml

```diff
@@ -71,4 +71,6 @@ is_variant = ["dep:convert_case"]
 unwrap = ["dep:convert_case"]
 try_unwrap = ["dep:convert_case"]
 
+std = []
```

Comment:
@JelteF I don't see why we need this feature here. We don't use any such `cfg`s inside the `-impl` crate.

Reply:
You're right, it's not needed anymore. It was from before I moved the error picking logic to the non-impl crate.